scan-payment/src/main/java/com/getbouncer/scan/payment/Image.kt

```diff
@@ -86,159 +94,182 @@ fun hasOpenGl31(context: Context): Boolean {
 /**
  * Fragments the image into multiple segments and places them in new segments.
  */
-fun Bitmap.fragment(
-    fromSegments: Array<Rect>,
-    toSegments: Array<Rect>,
-    toSize: Size
+fun Bitmap.rearrangeBySegments(
+    fromSegments: Map<Rect, Rect>,
+    futureImageSize: Size
 ): Bitmap {
-    require(fromSegments.size == toSegments.size) {
-        "Number of source segments does not match number of destination segments"
-    }
-
     val current = this
```

Comment:
remove this and replace with `this@Bitmap`

Reply:
@Bitmap is not required, only use this